config.ini

```diff
@@ -34,6 +34,10 @@ HTTPTimeout=2.5
 Username =
 Password =
 
+# Number of inverters if not all from json response should be used 
```

Comment:
Move it up to line 10, also call it differently to show, that we query only the first amount X of devices. QueryOnlyFirstOfDTUInverter

Comment should be like: If you want to exclude Inverter, specify how many of the Inverters you want to query. Please assure that the order is correct in the DTU, we can only extract the first one in a row. 

Please also update the README page